src/script/api/script_window.hpp.in

```diff
@@ -67,45 +67,49 @@ public:
 	 * Special number values.
 	 */
 	enum NumberType {
-		NUMBER_ALL = 0xFFFFFFFF, ///< Value to select all windows of a class.
+		NUMBER_ALL = -1, ///< Value to select all windows of a class.
 	};
 
 	/**
 	 * Special widget values.
 	 */
 	enum WidgetType {
-		WIDGET_ALL = 0xFF, ///< Value to select all widgets of a window.
+		WIDGET_ALL = -1, ///< Value to select all widgets of a window.
 	};
 
 	/**
 	 * Close a window.
 	 * @param window The class of the window to close.
 	 * @param number The number of the window to close, or NUMBER_ALL to close all of this class.
+	 *               The value will be clamped to 0 .. MAX(int32).
```

Comment:
`The value will be clamped to 0 .. MAX(int32) when value is not NUMBER_ALL` or something similar? Also for the other functions, including widget, so 4 cases in total.